### PR TITLE
Move Developer section and reference the contributing guide

### PR DESCRIPTION
**Pull request by jodh-intel**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,6 +43,10 @@ See https://github.com/clearcontainers/runtime/wiki/Installation.
 
 See https://github.com/hyperhq/runv.
 
+# Governance
+
+The Kata Containers project is governed according to the [“four opens"](https://governance.openstack.org/tc/reference/opens.html), which are open source, open design, open development and open community. Technical decisions will be made by technical contributors and a representative Architecture Committee. The community is committed to diversity, openness, encouraging new contributors and leaders to rise up.
+
 ## Developers
```
Comment (grahamwhaley) [inline README.md]: semi-unrelated nit: I thought I'd (asked to have) changed:

` who has merged code within the last 12 months`
to
`who has had code merged within the last 12 months`

Or was that in another doc/repo?

File: README.md [hunk 2]
```diff
@@ -5,6 +5,7 @@
     * [Users](#users)
```
Comment (grahamwhaley) [inline README.md]: in commit message s/the read/the reader/

File: README.md (hunk 1 above)
Comment (jodh-intel) [reply README.md]: Good catch - fixing.. Not clear on your question though I'm afraid?

**Comment by jodh-intel** [comment]
Branch updated.

File: README.md (hunk 1 above)
Comment (grahamwhaley) [reply README.md]: I thought I'd caught this before - maybe it was the same phrase in another doc though - np - thanks for fixing!

**Comment by jodh-intel** [comment]
Hi @klynnrif - could you take a look please?

File: README.md [hunk 3]
```diff
@@ -43,22 +44,26 @@ See https://github.com/clearcontainers/runtime/wiki/Installation.
 
 See https://github.com/hyperhq/runv.
 
+## Contributors
+
+See the [contributing guide](CONTRIBUTING.md) for details of how to contribute to the project.
```
Comment (klynnrif) [inline README.md]: ...for details on how to contribute to the project.

File: README.md [hunk 4]
```diff
@@ -43,22 +44,26 @@ See https://github.com/clearcontainers/runtime/wiki/Installation.
...
+
+# Governance
+
+The Kata Containers project is governed according to the [“four opens"](https://governance.openstack.org/tc/reference/opens.html), which are open source, open design, open development and open community. Technical decisions will be made by technical contributors and a representative Architecture Committee. The community is committed to diversity, openness, encouraging new contributors and leaders to rise up.
```
Comment (klynnrif) [inline README.md]: ...which are open source, open design, open development, and open community. Technical decisions are made by technical contributors and a representative Architecture Committee. The community is committed to diversity, openness, and encouraging new contributors and leaders to rise up.

**Comment by jodh-intel** [comment]
Thanks @klynnrif - branch updated.

**Pull request merged**